### ISSUE TITLE
Data information

### ISSUE THREAD

**Issue by ahmetdemirtas1** [neheller/kits23#17]
Are sample 1 and sample 2 used for right and left kidney?
imaging.nii.gz is the raw data. 
segmentation.nii.gz is masking information.
Is the raw data for masking and visualisation done with the same library?
Should the masking information be visualised using other methods?

**Comment by neheller** [comment]
You are correct that `imaging.nii.gz` is the raw image data, and `segmentation.nii.gz` is the full aggregated mask.

In the case where multiple 'instances' of a label exist in the dataset (such as two kidneys), they are denoted with 'instance-1', 'instance-2', etc. under the `instances/` folder. These were aggregated together to produce `segmentation.nii.gz` and can probably be safely ignored unless you're trying something unorthodox.

**Comment by AwayahShahid** [comment]
Hi, I cannot seem to find the imaging.nii.gz files
